src/js/browser-plugin.js

```diff
@@ -956,6 +970,7 @@ const createTweetLabel = (tweet, label, modules, callback) => {
   labelcat.append(txt);
 
   labelcat.addEventListener('click', (event) => {
+    log2Server('explainability', node.tweetUrl, `Tweet id: ${tweet.id}\nTweet label: ${node.coInformLabel}`, 'open label explainability popup');    
```

Comment:
I tested it and both node.tweetUrl and node.coInformLabel are undefined.
Please test and debug the different logs so each one is called when it should and the parameters are as expected.
I recommend to just set a breakpoint on line 1550 (inside function "log2Server"), and do the different actions that would generate the logs, and check what log is going to be generated at such code point.